### PR TITLE
[BD-26][EDUCATOR-5811] Add missed proctored exam instruction pages

### PR DESCRIPTION
**Pull request by UvgenGen**
**Description**: Add missed proctored exam instruction pages.

**Jira**: https://openedx.atlassian.net/browse/EDUCATOR-5811

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @UvgenGen! I've created [BLENDED-874](https://openedx.atlassian.net/browse/BLENDED-874) to keep track of it in Jira.   More details are on the [BD-26](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2523627980/BD-26+Proctoring+MFE+and+configuration) project page.


When this pull request is ready, tag your edX technical lead.

File: src/instructions/Instructions.test.jsx [hunk 1]
```diff
@@ -147,4 +147,74 @@ describe('SequenceExamWrapper', () => {
 
     expect(getByTestId('pending-prerequisites')).toBeInTheDocument();
   });
+
+  it('Instructions for error status', () => {
```
Comment (Zacharis278) [inline src/instructions/Instructions.test.jsx:151]: left a comment about this test in another PR, we should make sure to test the escalation/contact link as well. We can merge this for now and cover it in that test PR just to keep things moving.

File: src/instructions/proctored_exam/EntranceProctoredExamInstructions.jsx [hunk 2]
```diff
@@ -1,22 +1,42 @@
...
+            <div className="h3" data-testid="exam-instructions-title">
+              <FormattedMessage
+                id="exam.ReadyToResumeProctoredExamInstructions.title"
+                defaultMessage="Your exam is ready to be resumed."
+              />
+            </div>
+            <p>
+              <FormattedMessage
+                id="exam.ReadyToResumeProctoredExamInstructions.text"
+                defaultMessage="You will have 25 minutes to complete your exam."
```
Comment (Zacharis278) [inline src/instructions/proctored_exam/EntranceProctoredExamInstructions.jsx:26]: it looks like we just missed a hard coded number here, this value should be the time remaining for the attempt

File: src/instructions/Instructions.test.jsx [hunk 3]
```diff
@@ -147,4 +147,74 @@ describe('SequenceExamWrapper', () => {
...
+
+    render(
+      <ExamStateProvider>
+        <Instructions>
+          <div data-testid="sequence-content">Sequence</div>
+        </Instructions>
+      </ExamStateProvider>,
+      { store },
+    );
+    expect(screen.getByText('Your exam is ready to be resumed.')).toBeInTheDocument();
```
Comment (Zacharis278) [inline src/instructions/Instructions.test.jsx:217]: I think we should be testing for the correct "remaining time" value here. If you want to cover this in one of the test PRs that's fine by me

File: src/instructions/proctored_exam/ErrorProctoredExamInstructions.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            defaultMessage="Error with proctored exam"
+          />
+        </div>
+        <p className="mb-0">
+          <FormattedMessage
+            id="exam.ErrorProctoredExamInstructions.text"
+            defaultMessage={'A system error has occurred with your proctored exam. '
+            + 'Please reach out to '}
+          />
+          <Hyperlink href={contactUsUrl}>
```
Comment (Zacharis278) [inline src/instructions/proctored_exam/ErrorProctoredExamInstructions.jsx:27]: there should be two different cases for the contact link here depending on if there is a "proctoring_escalation_email" setup for the course team or not

**Review by Zacharis278 — CHANGES_REQUESTED** [review]

**Comment by ViktorRusakov** [comment]
@Zacharis278 I fixed the issues with hardcoded values, we will also have a PR in edx-proctoring for this since currently both remaining time and proctoring_escalation_email are not included in any of our APIs
we will fix tests in another PR

**Review by Zacharis278 — APPROVED** [review]

**Pull request merged**